src/java.desktop/share/classes/javax/swing/JScrollBar.java

```diff
@@ -753,11 +753,40 @@ public void stateChanged(ChangeEvent e)   {
         }
     }
 
-    // PENDING(hmuller) - the next three methods should be removed
+    /**
+     * Unlike most components, JScrollBar will derive the minimum size from
+     * the preferred size in one axis and a fixed minimum size in the other.
+     * Thus, it overrides {@code JComponent.setMinimumSize} contract
+     * that subsequent calls to getMinimumSize will return the
+     * same value as set in {@code JComponent.setMinimumSize}
```

Comment:
```suggestion
     * Unlike most components, {@code JScrollBar} derives the minimum size from
     * the preferred size in one axis and a fixed minimum size in the other.
     * Thus, it overrides {@code JComponent.setMinimumSize} contract
     * that subsequent calls to {@code getMinimumSize} will return the
     * same value as set in {@code JComponent.setMinimumSize}.
```

@prrace Should the first sentence use the present tense instead of the future?

There should be a period in the end of the sentence.

Should we add a paragraph saying, calling `setMinimumSize` has no effect?